stdlib/package.py

```diff
@@ -137,6 +139,176 @@ def __init__(
             shutil.rmtree(self.package_cache)
         os.makedirs(self.package_cache)
 
+    def is_empty(self) -> bool:
+        """Tests if the ``wrap_cache`` of this :py:class:`.Package` contains at least a single file.
+
+        :note: If the wrap cache contains only empty directories, this function returns ``False``.
+
+        :returns: ``True`` if the cache contains at least a single file, ``False`` otherwise.
+        """
+        for _, _, filenames in os.walk(self.wrap_cache):
+            if len(filenames):
+                return False
+        return True
+
+    def drain(self, *paths: str):
+        """Drain the current :py:class:`.Build`, moving files from its ``install_cache`` to this
```

Comment:
```suggestion
        """Drain the current :py:class:`.Build`, moving files from their ``install_cache`` to this
```
Same error for the next two functions, on the lines 195 and 234